src/hammer-vlsi/hammer_utils/lef_utils.py

```diff
@@ -82,8 +83,8 @@ def get_sizes(source: str) -> List[Tuple[str, float, float]]:
             if regex_search:
                 if found_size:
                     raise ValueError("Found two SIZE statements in MACRO block for {m}".format(m=in_macro))
-                width = float(regex_search.group(1))
-                height = float(regex_search.group(2))
+                width = Decimal(regex_search.group(1))
```

Comment:
Is `Decimal(x)` a superset of `float(x)`? I think this is true but just double checking

Reply:
I believe so - I'll double check. I assume your intent is to ensure there's no string that will cast to `float` but will not cast to `Decimal`?

Reply:
Yeah. I think it shouldn't be an issue, just wanted to check

Reply:
Appears to be the case: https://docs.python.org/3/library/decimal.html#decimal.Decimal